ui/app/src/state/epics/pathNavigatorTree.ts

```diff
@@ -166,8 +181,90 @@ export default [
       })
     ),
   // endregion
-  // region pathNavigatorFetchPathChildren
+  // region pathNavigatorTreeBulkBackgroundRefresh
   (action$, state$) =>
+    action$.pipe(
+      // TODO: There's no special handling for background vs visible. Should there be?
```

Comment:
There doesn't seem to be any previous handling for background vs visible (in comparison with pathNavigator, that is in the reducer). The pathNavTree state doesn't have any `isFetching` or something implemented for the loading state to display. Should we create a ticket for that?